### PR TITLE
WordCloud using adjectives voted & New design for yearbook

### PR DESCRIPTION
**Pull request by sohamvg**

### CODE REVIEW

File: myapp/templates/myapp/comment.html [hunk 1]
```diff
@@ -2,59 +2,151 @@
...
+		font-size: 15px;
+		color: #979497;
+		line-height: 1.2;
+		padding: 12px;
+		text-align: center;
+	}
+
+	.hiddenCB div {
+		display: inline;
+		margin: 0;
```
Comment (aman71197) [inline myapp/templates/myapp/comment.html]: Move this css to a static file

File: myapp/templates/myapp/otherComment.html [hunk 2]
```diff
@@ -37,7 +37,31 @@ <h6> from {{ name }} ({{ fromWhom }})</h6>
...
+				<hr>
+			{% endfor %}
+		</form>			
+	</div>
+	{% endif %}
+	<div>
+		<img style="height:200px;width: auto;max-width: 100%;" class="profile" src="{{wordcloud.url}}" alt="" />
+	</div>
+	===========================================================
+ -->
```
Comment (aman71197) [inline myapp/templates/myapp/otherComment.html]: Put it in a if block. Don't comment the code.

File: myapp/templates/myapp/comment.html (hunk 1 above)
Comment (sohamvg) [reply myapp/templates/myapp/comment.html]: Resolved in latest commit.

File: myapp/templates/myapp/otherComment.html (hunk 2 above)
Comment (sohamvg) [reply myapp/templates/myapp/otherComment.html]: Resolved in latest commit.

**Pull request merged**